src/about/newtab/newtab/tile.js

```diff
@@ -4,97 +4,80 @@
+export const update = (model:Model, action:Action):[Model, Effects<Action>] =>Unknown.update(model, action)
 
-const styleSheet = Style.createSheet
-  ( { tile:
-      { cursor: 'pointer'
-      , float: 'left'
-      , margin: '10px 25px 20px'
-      , width: '160px'
-      , textDecoration: 'none'
-      }
-    , image:
-      { backgroundColor: '#fff'
-      , backgroundSize: 'cover'
-      , backgroundPosition: 'center center'
-      , border: '1px solid rgba(0,0,0,0.15)'
-      , borderRadius: '12px'
-      , boxSizing: 'border-box'
-      , height: '100px'
-      , margin: '0 0 10px'
-      , width: '160px'
-      }
-    , title:
-      { fontSize: '14px'
-      , fontWeight: 'medium'
-      , lineHeight: '20px'
-      , overflow: 'hidden'
-      , textAlign: 'center'
-      , textOverflow: 'ellipsis'
-      , whiteSpace: 'nowrap'
-      , width: '100%'
-      }
-    , titleLight:
-      { color: 'rgba(0,0,0,0.8)'
-      }
-    , titleDark:
-      { color: 'rgba(255,255,255,0.7)'
-      }
-    }
-  );
+const styleSheet = Style.createSheet({
+                                       tile: {
+                                         cursor: 'pointer',
+                                         float: 'left',
+                                         margin: '10px 25px 20px',
+                                         width: '160px',
+                                         textDecoration: 'none'
+                                       }, image: {
+    backgroundColor: '#fff',
+    backgroundSize: 'cover',
+    backgroundPosition: 'center center',
+    border: '1px solid rgba(0,0,0,0.15)',
+    borderRadius: '12px',
+    boxSizing: 'border-box',
+    height: '100px',
+    margin: '0 0 10px',
+    width: '160px'
+  }, title: {
+    fontSize: '14px',
+    fontWeight: 'medium',
+    lineHeight: '20px',
+    overflow: 'hidden',
+    textAlign: 'center',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    width: '100%'
+  }, titleLight: {
+    color: 'rgba(0,0,0,0.8)'
+  }, titleDark: {
+    color: 'rgba(255,255,255,0.7)'
+  }
+                                     });
 
-export const view =
-  (model/*:Model*/, address/*:Address<Action>*/, isDark/*:boolean*/)/*:DOM*/ =>
-  html.a
-  ( { className: 'tile'
-    , style: styleSheet.tile
-    , target: '_blank'
-    , rel: 'noopener noreferrer'
-    , href: model.uri
-    }
-  , [ html.div
-      ( { className: 'tile-image'
-        , style: Style.mix
-            ( styleSheet.image,
-              { backgroundImage: `url(${model.src})`
-              }
-            )
-        }
-      )
-    , html.div
-      ( { className: 'tile-title'
-        , style: Style.mix
-          ( styleSheet.title
-          , ( isDark
-            ? styleSheet.titleDark
-            : styleSheet.titleLight
-            )
-          )
-        }
-      , [ model.title
-        ]
-      )
-    ]
-  );
+export const view = (model:Model, address:Address<Action>, isDark:boolean):DOM =>html.a({
```

Comment:
Yeah, I fully agree. To be able to work with automatic formatters we'll have to change code like this, though: without doing something like assigning intermediary results or, in this case, extracting type definitions, a formatter will always be too dumb to properly format this.